src/NodeApi/Interop/JSRuntimeContext.cs

```diff
@@ -656,9 +659,8 @@ private JSValue GetOrCreateObjectWrapper<T>(T obj, Func<JSValue> createWrapper)
                     return new JSReference(propertyValue);
                 }
             }
-
         });
-        return reference.GetValue() ?? JSValue.Undefined;
+        return reference.GetValue();
```

Comment:
It seems that the logic is changed here: we used to return `Undefined` if the reference has no value, and now it throws.

Reply:
It is always a strong reference so it could never have returned `Undefined` before.